python/ray/train/torch/torch_predictor.py

```diff
@@ -104,11 +104,25 @@ def _array_to_tensor(
     def _tensor_to_array(self, tensor: torch.Tensor) -> np.ndarray:
         return tensor.cpu().detach().numpy()
 
+    @DeveloperAPI
```

Comment:
In general, could we move underscore methods to be ordered after non-underscore methods? This helps with readability (non-underscore methods first).

Reply:
sure

Reply:
updated